lambda_fns/entry_predict/app.py

```diff
@@ -19,13 +21,18 @@
 RELIABILITY_FN_NAME = os.environ.get("RELIABILITY_FN_NAME")
 MODEL_INFO_FN_NAME = os.environ.get("MODEL_INFO_FN_NAME")
 
+SENTRY_URL = os.environ.get("SENTRY_URL")
+ENVIRONMENT = os.environ.get("ENVIRONMENT")
```

Comment:
Maybe use **django-environ** for env configuration and load all configurations from a single app like config.py?

https://django-environ.readthedocs.io/en/latest/
example: https://github.com/the-deep/server/blob/develop/deep/settings.py#L22-L85

Reply:
I think this is not that important to use.. just add an extra dependency. but moved all the envs to a config.py file.